tests/integration/test_prove.py

```
@@ -173,3 +174,61 @@ def test_prove_backend_filter_doom(self, model_name: str, model_dir: Path, slice
 
         assert found_slice_0, "slice_0 not found in execution_results"
         assert found_slice_1, "slice_1 not found in execution_results"
+
+    @pytest.mark.parametrize("model_name", ["doom"])
+    def test_prove_with_tiling(self, model_name: str, model_dir: Path, slices_output_dir: Path, run_output_dir: Path, capfd, jstprove_available):
```

Ruff (0.14.11)

Unused method argument: `model_name`

(ARG002)